bids/modeling/statsmodels.py

```diff
@@ -288,6 +288,17 @@ def __init__(self, level, name, transformations=None, model=None,
                 group_by.append(self.level)
         self.group_by = group_by
 
+        # Check for intercept only run level model and throw an error
+        try:
+            if (self.level == 'run') and (self.model['X'] == [1]):
```

Comment:
I think this is a field that (currently) gets lowercased. Could you add a test that makes sure this happens when we think?

```suggestion
            if (self.level == 'run') and (self.model['x'] == [1]):
```